torch/onnx/symbolic_opset9.py

```diff
@@ -1645,10 +1645,22 @@ def new_full(g, self, size, fill_value, dtype, layout, device, pin_memory=False)
     return full(g, size, fill_value, dtype, layout, device, pin_memory)
 
 
-def eye(g, n, m, dtype=None, layout=None, device=None, pin_memory=False):
-    shape = g.op("Concat", g.op("Unsqueeze", n, axes_i=[0]), g.op("Unsqueeze", m, axes_i=[0]), axis_i=0)
-    tensor = zeros(g, shape, dtype, layout, device)
-    return g.op("EyeLike", tensor)
+def eye(g, *args):
```

Comment:
It seems that it's now broken for keyword args. Is it expected?

Reply:
yes this is expected. It's not expected to see kwargs for symbolic functions, except for a few exceptions like `_outputs`.